scripts/post-install.sh

```diff
@@ -53,6 +50,12 @@ if [[ -f /etc/redhat-release ]]; then
         install_chkconfig
     fi
 elif [[ -f /etc/debian_version ]]; then
+    # Ownership for RH-based platforms is set in build.py via the `rmp-attr` option.
+    # We perform ownership change only for Debian-based systems.
+    # Moving these lines out of this if statement would make `rmp -V` fail after installation.
+    chown -R -L influxdb:influxdb $DATA_DIR
```

Comment:
The complemetary PRs https://github.com/influxdata/telegraf/pull/5716/ and https://github.com/influxdata/chronograf/pull/5146/ conditionally make the directories and `chmod 755` then. Does this need the same logic for the data and log dirs?

Reply:
I see your point. What I did here is to move the lines responsible for ownership change in order to make `rpm -V` pass and to leave the script's behavior the same as before when not using RHEL.  
It seems that the script didn't need those lines before.

Anyhow, I think I can add `chmod` lines without affecting the "correctness" of the script.